packages/core/js/MobileCore.js

```diff
@@ -160,15 +160,15 @@ module.exports = {
    * Dispatches a response event for a paired event that was sent to {@code dispatchEventWithResponseCallback}
    * and received by an extension listener {@code hear} method.
    *
-   * @param responseEvent required parameter, {@link AEPExtensionEvent} instance to be dispatched as a response for the
-   *                      event sent using {@link AEPCore#dispatchEventWithResponseCallback(AEPExtensionEvent)}
+   * @param responseEvent required parameter, {@link Event} instance to be dispatched as a response for the
+   *                      event sent using {@link AEPCore#dispatchEventWithResponseCallback(Event)}
    * @param requestEvent  required parameter, the event sent using
-   * 						{@link AEPCore#dispatchEventWithResponseCallback(AEPExtensionEvent)}
+   * 						{@link AEPCore#dispatchEventWithResponseCallback(Event)}
    * @return {@code boolean} indicating if the the event dispatching operation succeeded
    *
-   * @see AEPCore#dispatchEventWithResponseCallback(AEPExtensionEvent)
+   * @see AEPCore#dispatchEventWithResponseCallback(Event)
    */
-  dispatchResponseEvent(responseEvent: AEPExtensionEvent, requestEvent: AEPExtensionEvent): Promise<boolean> {
+  dispatchResponseEvent(responseEvent: Event, requestEvent: Event): Promise<boolean> {
```

Comment:
In Swift we don't have this API and you can register a listener + dispatch event, not sure about later versions of Android core, but  do we want to add this API in AEPCore?

Reply:
Let's discuss about this and make any changes in a new PR.